src/pages/wallet/AssetChain.tsx

```diff
@@ -129,18 +125,11 @@ const AssetChain = (props: Props) => {
                 {wrong ? (
                   <span className="danger">{t("Failed to query balance")}</span>
                 ) : (
-                  <Read
-                    {...props}
-                    amount={balance}
-                    token=""
-                    fixed={2}
-                    decimals={decimals}
-                  />
+                  <Read {...props} amount={balance} fixed={2} />
```

Comment:
Allow for IBCed token to display with native symbol:

<img width="1728" alt="Screenshot 2023-08-18 at 11 55 46 AM" src="https://github.com/terra-money/station/assets/30275692/a9bc4b85-d53d-42de-be95-f12321c112b5">

Reply:
fixed